src/components/skeleton/Header/Header.js

```diff
@@ -28,7 +29,9 @@ export default class Header extends Component {
     const t = this.context.t;
     const loginOrProfile = !this.props.user.playerId ?
       <FlatButton label={t('nav.login')} onClick={() => browserHistory.push(t.route('login'))} /> :
-      <Link className="Header-link Header-icon-right" to={t.route('profile')}><Icon fa="fa fa-2x fa-user" /></Link>;
+      <Link className="Header-link Header-icon-right" to={t.route('profile')}>
+          <Icon fa="fa fa-2x fa-user" />&nbsp;&nbsp;<b>{this._reverseName(storeUtil.getPlayer(this.props.user.playerId).name)}</b>
+      </Link>;
```

Comment:
heb je dit op mobile getest?
de reden dat er geen "TTC" voor Erembodegem staat is omdat het anders afgekapt wordt op mobile

met deze tekst erbij wordt het sowieso afgekapt


en functioneel gezien: ik denk dat iedereen zijn eigen naam wel weet zeker?